dynamics-nav/SETASCENDING-Function--Record-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETASCENDING Function (Record)"
+description: SETASCENDING Function (Record) sets the sort order for the records returned.
```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.